src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -21,7 +23,7 @@ def create_output_path(input_file, output_dir) -> str:
 
     def iter_paths(sequence) -> Iterator[Tuple]:
         for uri in sequence:
-            if uri.endswith((".tiff", ".tif", ".tdb")):
+            if uri.endswith(tuple([ext.value for ext in SupportedExtensions])):
```

Comment:
nit: [] not needed here (i.e., just `tuple(ext.value ...)`). Also I must have missed this last time I reviewed—I didn’t realize you could do `.endswith(('multiple', 'things'))`. Neat!